modin/error_message.py

```diff
@@ -48,8 +48,8 @@ def default_to_pandas(cls, message=""):
         if not cls.printed_request_implementation:
             message = (
                 "{}\n".format(message)
-                + "To request implementation, send an email to "
-                + "feature_requests@modin.org."
+                + "Please refer to "
```

Comment:
Given this message, `printed_request_implementation` should be corrected too.

Reply:
> Given this message, `printed_request_implementation` should be corrected too.

Fixed, I suppose

Reply:
fixed